package.json

```diff
@@ -48,9 +48,9 @@
     "@typescript-eslint/eslint-plugin": "^8.12.2",
     "@typescript-eslint/parser": "^8.12.2",
     "dotenv": "^16.3.1",
-    "eslint": "^9.13.0",
+    "eslint": "^9.14.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-header": "^3.1.1",
+    "eslint-plugin-headers": "^1.2.0",
```

Comment:
I had to use a different (claims to be successor) plugin as the previous one was no longer maintained, it seems, and it does not support the new flat ESLint config. I hope this plugin is OK and the license is compatible.

https://github.com/robmisasi/eslint-plugin-headers